### DISCUSSION TITLE
Allow setting the default language via ENV

### DISCUSSION THREAD

**Discussion by lahdekorpi** [directus/directus#5134] [Feature Requests]
Would like to be able to translate the login page as well and set the default language. Would it be enough to just have this set from an ENV variable if available? https://github.com/directus/directus/blob/3de49c373e70cd34e752c4d58bc7b16386740469/app/src/lang/index.ts#L13

**Comment by rijkvanzanten** [comment]
That's a good idea. Only downside is that the env vars are API-side, while we need this info in the app _before_ the Vue app is initialized 🤔 

I think directus_settings might be a better place for this, so we can fetch and set it immediately in the hydration of the project info (those are fetched first for public page login background etc)

**Comment by jesusgp22** [comment]
This is a must have for non-english users as they will get everything in english until they set their language on the admin page

**Reply by rijkvanzanten** [reply]
See https://github.com/directus/directus/discussions/7062#discussioncomment-1087252, wanna open a PR to add that in?

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
